src/components/ContextualMenu/ContextualMenu.scss

```diff
@@ -53,6 +53,17 @@ $ContextualMenu-iconWidth: 24px;
   }
 }
 
+a.ms-ContextualMenu-link {
+  text-rendering: auto;
+  color: initial;
+  letter-spacing: normal;
+  word-spacing: normal;
+  text-transform: none;
```

Comment:
That makes it's style equivalent to that of a button. It keeps everything consistant in terms of looks.